projects/batfish-common-protocol/src/main/java/org/batfish/storage/StorageUtils.java

```diff
@@ -0,0 +1,56 @@
+package org.batfish.storage;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.SortedSet;
+import org.batfish.common.BatfishException;
+import org.batfish.common.BfConsts;
+import org.batfish.common.util.CommonUtil;
+
+public class StorageUtils {
+
+  private final Path _containerLocation;
+
+  public StorageUtils(Path containerLocation) {
+    _containerLocation = containerLocation;
+  }
+
+  public Path resolvePath(Path basePath, String... dirs) {
+    if (dirs.length == 0) {
+      return basePath;
+    }
+    return basePath.resolve(String.join("/", dirs));
```

Comment:
There's an OS independent constant somewhere in Java for `/`. Something like File.Seperator or Path.Separator or something along those lines. Use that instead.